test/test_utils.py

```diff
@@ -247,6 +247,24 @@ def test_draw_segmentation_masks(colors, alpha, device):
     torch.testing.assert_close(out[:, overlap], interpolated_overlap, rtol=0.0, atol=1.0)
 
 
+def test_draw_keypoints_dtypes():
+    image_uint8 = torch.full((3, 100, 100), 0, dtype=torch.uint8)
```

Comment:
This image should not be just zeros, otherwise it will be easy to miss subtle bugs. This should be the same as for the other test i.e.:

```suggestion
    torch.randint(0, 256, size=(3, 100, 100), dtype=torch.uint8)
```

and in fact you'll see that there's a bug because the test will fail